src/main/kotlin/calculator/domain/Calculator.kt

```diff
@@ -0,0 +1,27 @@
+package calculator.domain
+
+class Calculator {
+
+    fun calculate(numbers: List<String>): Int {
+        var sum = 0
+        for (number in numbers) {
+            checkValidateNumber(number)
+            checkNegativeNumber(number)
+            sum += number.toInt()
+        }
+
+        return sum
+    }
+
+    private fun checkValidateNumber(string: String) {
+        try {
+            string.toInt()
+        } catch (e: NumberFormatException) {
+            throw RuntimeException("$string 은 올바른 숫자가 아닙니다.")
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        string.toIntOrNull() ?: throw RuntimeException("$string 은 올바른 숫자가 아닙니다.")
```